docs/execution-providers/ROCm-ExecutionProvider.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ROCm (AMD)
+description: Instructions to execute ONNX Runtime with the AMD ROCm execution provider
+parent: Execution Providers
+nav_order: 6
+redirect_from: /docs/reference/execution-providers/ROCm-ExecutionProvider
+---
+
+# ROCm Execution Provider
+{: .no_toc }
+
+The ROCm Execution Provider enables hardware accelerated computation on AMD ROCm-enabled GPUs. 
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Install
+
+Pre-built binaries of ONNX Runtime with ROCm EP are published for most language bindings. Please reference [Install ORT](../install).
+
+## Requirements
+
+
+|ONNX Runtime|ROCm|
+|---|---|
+|master|5.2.3|
```

Comment:
The default branch is now called "main"
If you saw this from another EP page, appreciate if you can update it there as well